build.gradle.kts

```diff
@@ -8,9 +11,14 @@ plugins {
 }
 
 plugins.withType<JavaPlugin> {
-    tasks.withType<GroovyCompile> {
-        sourceCompatibility = "1.7"
-        targetCompatibility = "1.7"
+    tasks {
+        withType<GroovyCompile> {
+            sourceCompatibility = "1.7"
+            targetCompatibility = "1.7"
+        }
+        withType<KotlinCompile> {
+            kotlinOptions.jvmTarget = "1.8"
```

Comment:
1.7 was chosen as it minimal working version for Gradle, change it only if it was changed there too

Reply:
I get it, but Scala – in the end – will be completely removed anyway.